models/topics.models.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../db/connection')
+
+exports.fetchAllTopics = async () => {
+    try {
+        const { rows } = await db.query(`SELECT * FROM topics;`)
+        if (rows.length === 0) {
+            return Promise.reject({
+                status: 404,
+                msg: 'No results found'
+            })
+        } else {
```

Comment:
Similar to the above comment this isnt something we need to check for as we can and should assume our database will always have some data within in.